cmd/operator/controller/start.go

```diff
@@ -161,6 +162,9 @@ func NewStartCommand() *cobra.Command {
 	command.Flags().BoolVar(&secureMetrics, "secure-metrics", false, "If set the metrics endpoint is served securely")
 	command.Flags().BoolVar(&enableHTTP2, "enable-http2", false, "If set, HTTP/2 will be enabled for the metrics and webhook servers")
 
+	command.Flags().StringVar(&pprofBindAddress, "pprof-bind-address", "0", "The address the pprof endpoint binds to. "+
+		"Use the port :6060. If not set, it will be 0 in order to disable the pprof server")
```

Comment:
Since this flag is passed from `controller.pprof.port`, to me the suggestion in the flag help to use port 6060 seems unnecessary.